### ISSUE TITLE
Gatsby Theme Issues

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/riverscapes-gatsby-theme#8]
- [x] Replacing `static/images/logo.png` does not get picked up as the sites log.
- [ ] Menus don't collapse after mouse out (See below)
- [x] `On this page` navigation component always shows, even if there are no items to list.
- [x] StoryCards don't redirect (see QRiS home page on `gatsby-docs` branch)
- [x] I like the new "lighter" code block style. Can you apply this same lighter style to big code blocks (i.e. using three backticks)
- [ ] Is the menu limited to a single dropdown of items only? Can we not have more than one menu level?

<img width="867" alt="Screenshot 2023-12-13 at 3 21 45 PM" src="https://github.com/Riverscapes/riverscapes-gatsby-theme/assets/8645379/7ae316eb-94cf-4b0e-975e-c0e5e167e15f">

**Comment by MattReimer** [comment]
>  I like the new "lighter" code block style. Can you apply this same lighter style to big code blocks (i.e. using three backticks)

No, but we can choose a different prismjs theme for all code blocks. We can revisit this later

**Comment by MattReimer** [comment]
Closing this in favour of #39
